core/cmd/local_client.go

```diff
@@ -60,6 +60,14 @@ func (cli *Client) RunNode(c *clipkg.Context) error {
 		logger.Warn("Chainlink is running in DEVELOPMENT mode. This is a security risk if enabled in production.")
 	}
 
+	pwd, err := passwordFromFile(c.String("password"))
+	if err != nil {
+		return cli.errorOut(fmt.Errorf("error reading password: %+v", err))
+	}
+	// Set the keystore password for CSA keys. This can be removed when we
```

Comment:
Because we are not using the keystore, we need to be able to store the keystore password in the config, which will allow the csa package to encrypt the key. This is intended to be temporary until we combine the keystores.

Reply:
:+1:

Reply:
Hmm. Can we instead set the password on the keystore itself?

Reply:
I believe the keystore is going to refactored in @RyanRHall's combine keystore PR, so I didn't want to put it on there in case it didn't exist anymore. 

@RyanRHall What do you think about this?

Reply:
that's fine to put it on the csa keystore for now. Either way, it will be getting refactored.

Reply:
So I had another look at this and the problem is that when you that `NewApplication` constructor only takes a `cli.Config` as an argument, so we have no other option to pass down the password from the CLI command to the service itself on initialization.

One option could be to set a function, `SetCSAPassphrase`, on the `Application` interface which will update the service with the password.

Thoughts? @samsondav @RyanRHall